src/search/search-index-new/storage/manager.ts

```diff
@@ -249,6 +249,9 @@ export class StorageManager implements ManageableStorage {
             coll = coll.reverse()
         }
 
+        // Returns the whole list object as lists have id in them.
+        if (collectionName === 'customLists') return coll.toArray()
```

Comment:
This is a hack, but I totally understand why you need it in your `CustomListStorage.prototype.fetchListNameSuggestions`. I think I will add a special flag to the StorageManager `suggest` method to allow returning of both the unique keys and, optionally, the assoc. PKs. Should be fairly simple to implement :) I will let you know when I get around to it.